tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/generateResultCli.ts

```diff
@@ -28,17 +36,18 @@ async function main() {
         printHelp();
         throw new Error(`logfile is empty`);
     }
-    if (taskname === undefined) {
+    if (taskName === undefined) {
         printHelp();
-        throw new Error(`taskname is empty`);
-    } else if (Object.values(AzureSDKTaskName).includes(taskname)) {
+        throw new Error(`taskName is empty`);
+    } else if (!Object.values(AzureSDKTaskName).includes(taskName)) {
         printHelp();
-        throw new Error(`invalid taskname`);
+        console.log("Current taskName is %s", taskName);
```

Comment:
use console.error()?
And as you are main function, could you just print the error and return directly?
Is there any specific reason that you need to throw here?

Reply:
if main function throw an error, the catch function at the end of the file will print error stack and error msg and then exit(1). We want to see the error stack when some functions which main called throw an error.